ztp/kube-compare-reference/compare.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+function read_dir() {
+for file in `ls $1`
+  do 
+    if [ -d $1"/"$file ]
+    then
+      read_dir $1"/"$file
+    else
+      echo $1"/"$file
+    fi
+done
+}
+
+
+read_dir renderedv1 |grep yaml  > rendered_file
+read_dir ../source-crs |grep yaml  > source_file
+
+for file1 in `cat source_file`
+do
+    for file2 in `cat rendered_file`
+    do
+      if [ "${file1##*/}" = "${file2##*/}" ]
+      then
+        diff -u $file1 $file2
+        printf "\n"
+        echo $file1 >> same_file
+      fi
+    done
+done
+
+
+for file in `cat same_file`
+do
+    sed -i "/${file##*/}/d" source_file
+done
+
+printf "\n\nThe following files exist in source-crs only, but not found in reference:\n"
+cat source_file
+
+rm -rf same_file rendered_file source_file
```

Comment:
In order to serve as a pass/fail for the CI in the future, we should make sure this script exits `0` only if there were no differences found and no extra files detected, and `1` if we saw any differences or missing files.